### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by arkpery**
## Description

As an open-source project, some documentation has to be done, such as: README.md.

Changed proposed in this pull request:
- Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +32,7 @@ Voici les principales technologies et framework utilisé dans le projet :
 - Docker
```
Comment (Fenniche-Ouahab) [inline README.md:33]: React

File: README.md [hunk 2]
```diff
@@ -29,7 +32,7 @@ Voici les principales technologies et framework utilisé dans le projet :
...
 
 
-## Partie Frontend du projet
+## Partie Front-end du projet
 
-Voici le lien vers la partie Frontend de notre projet :
-- [Backend URL Shortened](https://github.com/tcharles2201/back-end-url-shortened)
+Voici le lien vers la partie Front-end de notre projet :
```
Comment (Fenniche-Ouahab) [inline README.md]: Back-end

File: README.md [hunk 3]
```diff
@@ -29,7 +32,7 @@ Voici les principales technologies et framework utilisé dans le projet :
 - Docker
```
Comment (Fenniche-Ouahab) [inline README.md:33]: React and Chakra UI

**Review by Fenniche-Ouahab — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (arkpery) [reply README.md:33]: Some changes has been done.

File: README.md (hunk 2 above)
Comment (arkpery) [reply README.md]: Some changes has been done.

**Pull request merged**